mifosng-android/src/main/java/com/mifos/mifosxdroid/online/savingaccountsummary/SavingsAccountSummaryFragment.java

```diff
@@ -456,4 +486,100 @@ public void onDestroyView() {
         void doTransaction(SavingsAccountWithAssociations savingsAccountWithAssociations, String
                 transactionType, DepositType accountType);
     }
+
+    // Takes ScreenShot of Relative layout
+    @RequiresApi(api = Build.VERSION_CODES.KITKAT)
```

Comment:
This app currently supports minimum sdk level of `15` but this function has some part which requires `19/+ sdk` so please handle these kind of cases to avoid app crash and feature unavailability in certain devices.